package.json

```
@@ -13,65 +13,65 @@
 		"docker:jt-up": "[ -f ./tools/docker/bin/jt/tunnel.sh ] || { echo 'Tunneling scripts are not installed. See the section \"Jurassic Tube Tunneling Service\" in tools/docker/README.md.'; exit; }; ./tools/docker/bin/jt/tunnel.sh",
-		"docker:multisite-convert": "yarn docker:compose exec wordpress bash -c \"/var/scripts/multisite-convert.sh\"",
-		"docker:ngrok": "yarn docker:compose -f tools/docker/docker-compose-ngrok.yml",
-		"docker:ngrok-down": "yarn docker:ngrok down",
-		"docker:ngrok-up": "yarn docker:ngrok up",
-		"docker:phpunit": "yarn docker:compose exec wordpress phpunit --configuration=/var/www/html/wp-content/plugins/jetpack/phpunit.xml.dist",
-		"docker:phpunit:multisite": "yarn docker:compose exec wordpress phpunit --configuration=/var/www/html/wp-content/plugins/jetpack/tests/php.multisite.xml",
-		"docker:run-extras": "yarn docker:compose exec wordpress bash -c \"chmod +x /var/scripts/run-extras.sh && . /var/scripts/run-extras.sh\"",
-		"docker:sh": "yarn docker:compose exec wordpress bash",
-		"docker:stop": "yarn docker:compose stop",
-		"docker:tail": "yarn docker:compose exec wordpress bash -c \"/var/scripts/tail.sh\"",
-		"docker:uninstall": "yarn docker:compose exec wordpress bash -c \"/var/scripts/uninstall.sh\"",
-		"docker:up": "yarn docker:compose up",
-		"docker:update-core-unit-tests": "yarn docker:compose exec wordpress svn up /tmp/wordpress-develop/tests/phpunit/data/ /tmp/wordpress-develop/tests/phpunit/includes",
-		"docker:wp": "yarn docker:compose exec wordpress wp --allow-root --path=/var/www/html/",
-		"install-if-deps-outdated": "yarn install --check-files --production=false --frozen-lockfile",
-		"lint": "yarn lint-file .",
+		"docker:multisite-convert": "pnpm run docker:compose -- exec wordpress bash -c \"/var/scripts/multisite-convert.sh\"",
+		"docker:ngrok": "pnpm run docker:compose -- -f tools/docker/docker-compose-ngrok.yml",
+		"docker:ngrok-down": "pnpm run docker:ngrok down",
+		"docker:ngrok-up": "pnpm run docker:ngrok up",
+		"docker:phpunit": "pnpm run docker:compose -- exec wordpress phpunit --configuration=/var/www/html/wp-content/plugins/jetpack/phpunit.xml.dist",
+		"docker:phpunit:multisite": "pnpm run docker:compose -- exec wordpress phpunit --configuration=/var/www/html/wp-content/plugins/jetpack/tests/php.multisite.xml",
+		"docker:run-extras": "pnpm run docker:compose -- exec wordpress bash -c \"chmod +x /var/scripts/run-extras.sh && . /var/scripts/run-extras.sh\"",
+		"docker:sh": "pnpm run docker:compose exec wordpress bash",
+		"docker:stop": "pnpm run docker:compose stop",
+		"docker:tail": "pnpm run docker:compose exec wordpress bash -c \"/var/scripts/tail.sh\"",
+		"docker:uninstall": "pnpm run docker:compose -- exec wordpress bash -c \"/var/scripts/uninstall.sh\"",
+		"docker:up": "pnpm run docker:compose up",
```

Should it be `pnpm run docker:compose -- up` here?

```suggestion
		"docker:up": "pnpm run docker:compose -- up",
```

I used to run `yarn docker:up -d` to have my container running in the background, but this now only works if I do `pnpm run docker:compose -- up -d`.

---

Hmm. Yeah, we need the `--` in a lot of places to make sure the calls go through correctly. More than just here.

---

9527526